cheatsheets/Content_Security_Policy_Cheat_Sheet.md

```diff
@@ -139,18 +158,20 @@ If the [upgrade-insecure-requests](https://developer.mozilla.org/en-US/docs/Web/
 
 - To prevent all framing of your content use:
   - `Content-Security-Policy: frame-ancestors 'none';`
-- To allow for your site only, use:
+- To allow for the site itself, use:
   - `Content-Security-Policy: frame-ancestors 'self';`
-- To allow for trusted domain , do the following:
+- To allow for trusted domain, do the following:
   - `Content-Security-Policy: frame-ancestors trusted.com;`
 
-## Loading Files from CDN
+## Strict Policy
+
+In order to have a locked down CSP policy, applying [nonces](https://en.wikipedia.org/wiki/Cryptographic_nonce) on all loaded javascript is a must.
```

Comment:
It would be nice if you could give two examples for a strict CSP that addresses the classical stored/reflected XSS:
Maybe copy the two from: https://speakerdeck.com/lweichselbaum/csp-a-successful-mess-between-hardening-and-mitigation?slide=55
First is easy to adopt, second is harder but has better coverage for some DOM XSS sinks.